### PR TITLE
option renaming

### PR DESCRIPTION
**Pull request by sophieberkhout**
New try for the option renaming :)

Classical and Bayesian meta-analysis (https://github.com/jasp-stats/INTERNAL-jasp/issues/2019, https://github.com/jasp-stats/INTERNAL-jasp/issues/2014).

### CODE REVIEW

**Comment by sophieberkhout** [comment]
Hmm it looks like the Windows unit tests are failing for the classical prediction performance, but I did not touch any of those files...

**Comment by Kucharssim** [comment]
It seems the tests have been failing for a while (https://jasp-stats.github.io/actions-dashboard/), perhaps @FBartos can take a look what's going wrong in there?

File: inst/qml/Upgrades.qml [hunk 1]
```diff
@@ -0,0 +1,148 @@
+import QtQuick		2.12
+import JASP.Module	1.0
+
+Upgrades
+{
+	Upgrade
+	{
+		functionName:	"ClassicalMetaAnalysis"
+		fromVersion:	"0.15"
```
Comment (Kucharssim) [inline inst/qml/Upgrades.qml:9]:
```suggestion
		fromVersion:	"0.16.4"
```

File: inst/qml/Upgrades.qml [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+		toVersion:		"0.17"
+
+		// ClassicalMetaAnalysis.qml
+		ChangeRename { from: "dependent"; to: "effectSize" }
+		ChangeRename { from: "wlsWeights"; to: "effectSizeStandardError" }
```
Comment (Kucharssim) [inline inst/qml/Upgrades.qml:14]:
```suggestion
		ChangeRename { from: "wlsWeights"; to: "effectSizeSe" }
```
as per the naming guide

File: inst/qml/Upgrades.qml [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+		ChangeRename { from: "studyLabels"; to: "studyLabel" }
+		ChangeRename { from: "includeConstant"; to: "includeIntercept" }
```
Comment (Kucharssim) [inline inst/qml/Upgrades.qml:16]:
```suggestion
		ChangeRename { from: "includeConstant"; to: "interceptTerm" }
```
to be consistent with other analyses

File: inst/qml/Upgrades.qml [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+		ChangeRename { from: "plotResidualsPredicted"; to: "profilePlot" }
+		ChangeRename { from: "plotResidualsDependent"; to: "diagnosticPlot" }
+		ChangeRename { from: "plotResidualsQQ"; to: "diagnosticQqPlot" }
+		ChangeRename { from: "plotResidualsCovariates"; to: "failSafeN" }
+		ChangeRename { from: "residualsCasewiseDiagnostics"; to: "casewiseDiagnostics" }
+	}
+	Upgrade
+	{
+		functionName:	"BayesianMetaAnalysis"
+		fromVersion:	"0.15"
```
Comment (Kucharssim) [inline inst/qml/Upgrades.qml:40]:
```suggestion
		fromVersion:	"0.16.4"
```

File: inst/qml/Upgrades.qml [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+		toVersion:		"0.17"
+
+		// BayesianMetaAnalysis.qml
+		ChangeRename { from: "standardError"; to: "effectSizeStandardError" }
```
Comment (Kucharssim) [inline inst/qml/Upgrades.qml:44]:
```suggestion
		ChangeRename { from: "standardError"; to: "effectSizeSe" }
```

File: inst/qml/Upgrades.qml [hunk 6]
```diff
@@ -0,0 +1,148 @@
+import QtQuick		2.12
```
Comment (Kucharssim) [inline inst/qml/Upgrades.qml:1]: The `Upgrades.qml` file should be under `inst/`, not `inst/qml/`... please move it there :)

File: inst/qml/Upgrades.qml [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+								case "labelForest":	        return "rowOrder";
+						}
+				}
+		}
+		ChangeRename { from: "plotCumForest"; to: "cumulativeForestPlot" }
+		ChangeRename { from: "addPrior"; to: "cumulativeForestPlotPrior" }
+		ChangeRename { from: "plotPosterior"; to: "priorAndPosterior" }
+		ChangeRename { from: "addInfo"; to: "priorAndPosteriorInfo" }
+		ChangeRename { from: "addLines"; to: "priorAndPosteriorFixedAndRandom" }
+		ChangeRename { from: "shade"; to: "priorAndPosteriorShade" }
```
Comment (Kucharssim) [inline inst/qml/Upgrades.qml:116]:
```suggestion
		ChangeRename { from: "plotPosterior"; to: "priorPosterior" }
		ChangeRename { from: "addInfo"; to: "priorPosteriorAdditionalInfo" }
		ChangeRename { from: "addLines"; to: "priorPosteriorFixedAndRandom" }
		ChangeRename { from: "shade"; to: "priorPosteriorCi" }
```

File: inst/qml/Upgrades.qml [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+		ChangeRename { from: "plotSequential"; to: "sequentialPlotBayesFactor" }
+		ChangeRename { from: "plotSeqPM"; to: "sequentialPlotModelProbability" }
```
Comment (Kucharssim) [inline inst/qml/Upgrades.qml:118]:
```suggestion
		ChangeRename { from: "plotSequential"; to: "bfSequentialPlot" }
		ChangeRename { from: "plotSeqPM"; to: "modelProbabilitySequentialPlot" }
```

**Review by Kucharssim — CHANGES_REQUESTED** [review]
Looks great, if you change the versions in the Upgrade file and move the file into `inst/`, the upgrades work!

I left some comments to improve consistency with other analyses. Please change those.

The tests fail in the other analyses, not in yours. @FBartos will fix those. When you implement my suggestions, we can merge this!

File: inst/qml/Upgrades.qml (hunk 1 above)
Comment (sophieberkhout) [reply inst/qml/Upgrades.qml]: Hi @Kucharssim, thank you for the suggestions. I have updated the files accordingly.

**Review by Kucharssim — APPROVED** [review]

**Comment by Kucharssim** [comment]
Merging this now, @FBartos please don't forget to make the PR that fixes the rest of the tests! Thanks.

**Pull request merged**